agile-development/intro-to-agile.md

```diff
@@ -54,14 +54,14 @@ The benefits of Agile software development includes:
 - Minimum Viable Product (MVP)
 
 ### Materials
-- [Manifesto for Agile Software Development](http://agilemanifesto.org/)
```

Comment:
Can you please put this back? It’s important, even if we mention it elsewhere

Reply:
Done!